### PR TITLE
Add xmllint and scss-lint

### PR DESCRIPTION
**Pull request by janjaapdriessen**
What are the chances of this pull request being accepted? If so, I'll write some tests.

### CODE REVIEW

**Comment by janjaapdriessen** [comment]
I'm going to add Chameleon-xmllint too. We found added value in checking that our chameleon templates are valid XML, we would need to add a HTML wrapper in some cases, as seen in p01.checker:

http://pydoc.net/Python/p01.checker/0.5.2/p01.checker.checker/

File: plone/recipe/codeanalysis/xmllint.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        # Please the ABC by faux-implementing the cmd.
+        pass
+
+    def run(self):
+        files = []
+        for extension in self.extensions:
+            files.extend(self.find_files('.*\.{0}'.format(extension)))
+
+        total_errors = []
+        for file in files:
```
Comment (gforcada) [inline plone/recipe/codeanalysis/xmllint.py]: ``file`` is a python reserved word, better use something else.

**Comment by gforcada** [comment]
:+1: for merging it, some documentation about what are they purpose would be good. A paragraph for each of them on the README is probably enough, and, yes the tests :-)

**Comment by janjaapdriessen** [comment]
Thanks @gforcada for the quick reply. I'll polish docs and tests now I know there is a chance of landing these changes upstream.

**Comment by tisto** [comment]
Hey @janjaapdriessen, as @gforcada said, add tests and docs and we are happy to review and merge...

**Comment by hvelarde** [comment]
a squash would be appreciated also.

**Comment by gforcada** [comment]
@hvelarde let him finish first though...

**Comment by jensens** [comment]
this one fails in Python 3.4 and is therefore unmerged, or what else is missing?

**Comment by jensens** [comment]
Hence there was no activity for a long time, I propose the to close this PR within next two weeks. If you do not feel ok with this, please speak up and provide us a rough schedule.

**Pull request closed without merging**